spec/visitorSpec.js

```diff
@@ -0,0 +1,20 @@
+visit = require("../src/visitor");
+var fs = require('fs');
+
+describe("spies", function () {
+  var visitor = new Visitor("bob", "smith", 2 / 1 / 2019, 7, "sally");
+  it('should spy on save', function () {
+    var spy = spyOn(visitor, 'save');
+    visitor.save();
+    expect(spy).toHaveBeenCalled();
+  })
+})
+
+
+describe("read", function () {
+  var visitor = new Visitor("alice", "mere", 12 / 07 / 2019, 6, "josh");
```

Comment:
The load function call fails because the file visitor1.json does not exist. Lets work together on this. Please ping me.

Reply:
Good day Rudy,i did the changes on my work,thank you.